exercises/allergies/src/test/scala/AllergiesTest.scala

```diff
@@ -1,104 +1,104 @@
+  }
+
+  test("Allergen.Peanuts - allergic to strawberries but not peanuts") {
+    pending
+    Allergies.allergicTo(Allergen.Peanuts, 9) should be (false)
+  }
+
+  test("Allergen.Shellfish - allergic to strawberries but not peanuts") {
+    pending
+    Allergies.allergicTo(Allergen.Shellfish, 9) should be (false)
+  }
+
+  test("Allergen.Strawberries - allergic to strawberries but not peanuts") {
+    pending
+    Allergies.allergicTo(Allergen.Strawberries, 9) should be (true)
   }
 
   test("no allergies at all") {
     pending
-    Allergies.list(0) should be(List())
+    Allergies.list(0) should be (List())
   }
 
   test("allergic to just eggs") {
     pending
-    Allergies.list(1) should be(List(Allergen.Eggs))
+    Allergies.list(1) should be (List(Allergen.Eggs))
   }
 
   test("allergic to just peanuts") {
     pending
-    Allergies.list(2) should be(List(Allergen.Peanuts))
+    Allergies.list(2) should be (List(Allergen.Peanuts))
   }
 
   test("allergic to just strawberries") {
     pending
-    Allergies.list(8) should be(List(Allergen.Strawberries))
+    Allergies.list(8) should be (List(Allergen.Strawberries))
   }
 
   test("allergic to eggs and peanuts") {
     pending
-    Allergies.list(3) should be(List(Allergen.Eggs, Allergen.Peanuts))
+    Allergies.list(3) should be (List(Allergen.Eggs, Allergen.Peanuts))
   }
 
   test("allergic to more than eggs but not peanuts") {
     pending
-    Allergies.list(5) should be(List(Allergen.Eggs, Allergen.Shellfish))
+    Allergies.list(5) should be (List(Allergen.Eggs, Allergen.Shellfish))
   }
 
   test("allergic to lots of stuff") {
     pending
-    Allergies.list(248) should be(
-      List(Allergen.Strawberries,
-           Allergen.Tomatoes,
-           Allergen.Chocolate,
-           Allergen.Pollen,
-           Allergen.Cats))
+    Allergies.list(248) should be (List(Allergen.Strawberries, Allergen.Tomatoes, Allergen.Chocolate, Allergen.Pollen, Allergen.Cats))
   }
 
   test("allergic to everything") {
     pending
-    Allergies.list(255) should be(
-      List(Allergen.Eggs,
-           Allergen.Peanuts,
-           Allergen.Shellfish,
-           Allergen.Strawberries,
-           Allergen.Tomatoes,
-           Allergen.Chocolate,
-           Allergen.Pollen,
-           Allergen.Cats))
+    Allergies.list(255) should be (List(Allergen.Eggs, Allergen.Peanuts, Allergen.Shellfish, Allergen.Strawberries, Allergen.Tomatoes, Allergen.Chocolate, Allergen.Pollen, Allergen.Cats))
   }
 
   test("ignore non allergen score parts") {
     pending
-    Allergies.list(509) should be(
-      List(Allergen.Eggs,
-           Allergen.Shellfish,
-           Allergen.Strawberries,
-           Allergen.Tomatoes,
-           Allergen.Chocolate,
-           Allergen.Pollen,
-           Allergen.Cats))
+    Allergies.list(509) should be (List(Allergen.Eggs, Allergen.Shellfish, Allergen.Strawberries, Allergen.Tomatoes, Allergen.Chocolate, Allergen.Pollen, Allergen.Cats))
```

Comment:
The previous version was a bit more readable, but I assume this is just default behavior of the test generator?